CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
 # Change Log
 
 ## [Unreleased]
+### Added
+- Deserializer - Expose Deserializer module to API
+
+## RELEASE 0.1.31 - 2016-11-17
 ### Fixed
 - Custom Actions - Fix missing actions for Smart Collections.
```

Comment:
This fix has not been released yet, I'll fix the Changelog in the master branch.